### PR TITLE
Ctcp 4328

### PR DESCRIPTION
**Pull request by derekspalmer**

### CODE REVIEW

File: it/migrations/MovementsChangeLogSpec.scala [hunk 1]
```diff
@@ -28,103 +28,103 @@ import org.scalatest.BeforeAndAfterAll
...
-            arrival.messages.mapWithIndex {
-              case (message, index) =>
-                message.messageId.value mustBe (index + 1)
-            }
-        }
-      }
-    }
-  }
-}
+//class MovementsChangeLogSpec { // extends ItSpecBase with IntegrationPatience with BeforeAndAfterAll with GuiceOneAppPerSuite {
```
Comment (brianbayes) [inline it/migrations/MovementsChangeLogSpec.scala:40]: Accidental commented out code?

File: app/utils/IndexUtils.scala [hunk 2]
```diff
@@ -16,42 +16,42 @@
...
-    min = None,
-    max = None,
-    bucketSize = None,
-    collation = None,
-    wildcardProjection = None,
-    version = None,
-    partialFilter = None,
-    options = options
-  )
+//  def index(
```
Comment (brianbayes) [inline app/utils/IndexUtils.scala:27]: Accidental commented out code?

File: app/testOnly/controllers/TestOnlyController.scala [hunk 3]
```diff
@@ -45,23 +41,36 @@ class TestOnlyController @Inject()(
...
+          .toFuture()
+          .map(
+            _ => Ok(s"Cleared '${mongo.collectionName}' Mongo collection")
           )
       } else {
-        Future.successful(NotImplemented(s"Feature disabled, cannot drop ${ArrivalMovementRepository.collectionName}"))
+        Future.successful(NotImplemented(s"Feature disabled, cannot drop ${mongo.collectionName}"))
       }
   }
+//  Action.async {
```
Comment (brianbayes) [inline app/testOnly/controllers/TestOnlyController.scala:54]: Accidental commented out code?

File: app/repositories/ArrivalMovementRepository.scala [hunk 4]
```diff
@@ -315,210 +312,135 @@ class ArrivalMovementRepository @Inject()(
...
-      Json.obj(
-        "messages" -> Json.obj(
-          "$elemMatch" -> Json.obj(
-            "messageJson" -> Json.obj()
-          )
-        )
-      )
-
-    val query = Json.obj("$or" -> Json.arr(messagesWithNoJson, messagesWithEmptyJson))
+    // TODO: is this correct?
```
Comment (brianbayes) [inline app/repositories/ArrivalMovementRepository.scala:428]: Does this need looking or is it just a redundant comment?

**Review by brianbayes — COMMENTED** [review]
Looks like we have some commented-out code that could do with being removed.

File: it/migrations/MovementsChangeLogSpec.scala (hunk 1 above)
Comment (derekspalmer) [reply it/migrations/MovementsChangeLogSpec.scala:40]: Not accidental as such but is not required and will be removed.

File: app/utils/IndexUtils.scala (hunk 2 above)
Comment (derekspalmer) [reply app/utils/IndexUtils.scala:27]: Yep - no longer require and will be removed.

File: app/testOnly/controllers/TestOnlyController.scala (hunk 3 above)
Comment (derekspalmer) [reply app/testOnly/controllers/TestOnlyController.scala:54]: Yep - to be removed.

File: app/repositories/ArrivalMovementRepository.scala (hunk 4 above)
Comment (derekspalmer) [reply app/repositories/ArrivalMovementRepository.scala:428]: Not sure if we need this anymore - need to speak to frontend to see if they call it.

**Pull request closed without merging**